src/api/subscriptions.js

```diff
@@ -208,6 +214,10 @@ export const SubscriptionFlows = {
 };
 
 
+/** @type {string} */
+export const LOGIN_WAITING_VIEW = 'loginWaitingView';
```

Comment:
I'd say we should either put it inside the "SubscriptionFlows" above or remove it completely. I don't think it's a useful analytical signal b/c publishers would always call this function themselves - i.e. they always aware when it's used.

Reply:
done